preamble/syllabus.qmd

```diff
@@ -4,152 +4,84 @@
-    courses online, but has no formal training in programming. Amira
-    would like to tidy up the scripts, data sets, and reports she has
-    created in order to share them with her colleagues. These lessons
-    will show her how to do this and what "done" looks like.
-
-Jun
-
-:   Completed an [Insight Data
-    Science](https://insightfellows.com/data-science) fellowship last
-    year after doing a PhD in Geology and now works for a company that
-    does forensic audits. He uses a variety of machine learning and
-    visualization packages, and would now like to turn some of his own
-    work into an open source project. This book will show him how such a
-    project should be organized and how to encourage people to
-    contribute to it.
-
-Sami
-
-:   Became a competent programmer during a bachelor's degree in applied
-    math and was then hired by the university's research computing
-    center. The kinds of applications they are being asked to support
-    have shifted from fluid dynamics to data analysis; this guide will
-    teach them how to build and run re-usable data analysis pipelines so
-    that they can pass those skills on to their users.
-
-Ali
-
-:   Learned about statistics and programming during high school and
-    decided to enroll into an undergraduate Data Science program. She
-    hopes to have a career working on open source projects and doing
-    research software engineering for a research institution. One of her
-    courses uses this book so she will be working through the material
-    during the course and will use it as reference afterwards.
-
-Luis
-
-:   Has been doing a graduate degree that involves largely doing
-    quantitative research. While doing his projects, he realizes that
-    learning and creating R packages would greatly help his work as well
-    as others in the group. However, he has not had any formal courses
-    in computer programming. He enrolled in a course at the university
-    that uses this book so will work through this book.
-
-### Assumed skills and knowledge
+We designed the course with the following characteristics of learners
+like you in mind as the type of people who would be interested in and
+want to take this course. We **don't** assume learners to have all or
+even most of these characteristics, we only use these as a guide for
+structuring and developing this material as well as to help manage your
+expectations of the course.
+
+-   Uses [R](https://cran.r-project.org/) for doing statistical
+    analysis, general data analysis, data processing, data science, or
+    for other research purposes.
+-   Has no or limited formal training in programming or software
+    development.
+-   Has taken some formal R courses (like one of our previous
+    [ones](https://r-cubed.rostools.org)) and feels fairly competent in
+    writing R code.
+-   Writes a lot of R scripts and some functions, often with a fair
+    amount of repetitive code used throughout.
```

Comment:
Do we assume that learners *often* repeat code? It doesn't really fit with the assumptions further down that they use function-workflows (L81-86) ?

Reply:
Using a function-based workflow isn't a binary state, there is lots of repetition you can still do as you progress to getting better.